### PR TITLE
New quickstart UI and menu

### PR DESCRIPTION
**Pull request by nunix**
This PR changes the Quickstart menu and adds a new Quickstart UI page.

The menu shows the two options to install and use Elemental:

* First option is through Rancher Manager with the Elemental plugin
* Second option is with the CLI only

To avoid duplicates of common parts between the two UIs, the `partial` feature from Docusaurus is used.

This allows to write one portion of documentation into one page only, and import the page into both Quickstart pages.

Finally, the Overview has also been updated with the description of what's the Elemental stack. This is best fit there, instead of the Quickstart.

### CODE REVIEW

**Comment by kkaempf** [comment]
Assuming that this only touches the 'Next' version, I'm in favor of merging it asap and fix stuff iteratively.

**Review by davidcassany — COMMENTED** [review]
This looks good to me. Probably only small remark I'd do is that the images reference the already stable version, `Next` is currently targeting 1.2.x on the UI front. 

As Klaus suggested we can easily iterate afterwards.

File: docs/partials/_quickstart-prereqs.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+## Prerequisites
+
+* A Rancher server (v2.7.0) configured (server-url set)
+  * To configure the Rancher `server-url` please check the [Rancher docs](https://rancher.com/docs/rancher/v2.6/en/admin-settings/#first-log-in)
+* A machine (bare metal or virtualized) with TPM 2.0
+  * Hint 1: Libvirt allows setting virtual TPMs for virtual machines [example here](tpm/#add-tpm-module-to-virtual-machine)
+  * Hint 2: You can enable TPM emulation on bare metal machines missing the TPM 2.0 module [example here](tpm/#add-tpm-emulation-to-bare-metal-machine)
+  * Hint 3: Make sure you're using UEFI (not BIOS) on x86-64, or the ISO won't boot
+* Helm Package Manager (https://helm.sh/)
+* Docker (for ISO manipulation)
```
Comment (kkaempf) [inline docs/partials/_quickstart-prereqs.md:10]: Docker isn't required when using Rancher manager

File: docs/partials/_quickstart-prereqs.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+* For ARM - One SD-card (32 GB or more, must be **fast** - 40MB/s write speed is acceptable) and a USB-stick for installation
```
Comment (kkaempf) [inline docs/partials/_quickstart-prereqs.md:11]: I'm still not sure if we should name it "ARM" or "aarch64" or Raspberry Pi 😕

**Review by fgiudici — APPROVED** [review]
Great adoption the _partial_ feature  here 💚 
LGTM! We can start merging and extend with the SeedImage thing later.

File: docs/partials/_quickstart-prereqs.md (hunk 1 above)
Comment (nunix) [reply docs/partials/_quickstart-prereqs.md]: Removed `Docker` reference from the prerequisites

File: docs/partials/_quickstart-prereqs.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+## Prerequisites
+
+* A Rancher server (v2.7.0) configured (server-url set)
```
Comment (fgiudici) [inline docs/partials/_quickstart-prereqs.md:3]: nit: the latest UI with the latest ISO build feature requires Rancher 2.7.2 (which supports the latest version of the rancher ui extension operator, needed by the elemental UI extension 1.2.0).

File: docs/images/quickstart-ui-elemental-plugin-menu.png
Comment (fgiudici) [inline docs/images/quickstart-ui-elemental-plugin-menu.png:1]: nit: the OS Management logo changed in 1.2.0  (the new one is waaaayyyy cooler 😎 🤣  ).

File: docs/quickstart-ui.md [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+Remove the existing content and enter this code snippet and click on save. Remember to modify this as per your need. Click the `Create` button.
+
+![Create a Registration Endpoint with YAML](images/quickstart-ui-registration-endpoint-create-yaml.png)
+
+:::info main options
+`name: elemental-cluster1`: change this as per your need
+
+`device: /dev/sda`: make sure your target device is "sda". Otherwise find out how the disk device is named and change it here. For example, in Raspberry Pi it could be "mmblk"
+
+`emulate-tpm: true`: use this only if your target device doesn't have a TPM device and you have a way of emulating TPM like in VMware or KVM
```
Comment (fgiudici) [inline docs/quickstart-ui.md:113]: nit: I would not use tpm emulation in the UI quickstart... ok to remove it later (we should also update the MachineRegistration image)

File: docs/quickstart-ui.md [hunk 5]
```diff
@@ -0,0 +1,124 @@
...
+
+`emulated-tpm-seed: 1`: increase this by 1 for every new machine. If this value is not set, each machine will receive the same TPM hash and it won't show up under the machine inventory.
```
Comment (fgiudici) [inline docs/quickstart-ui.md:115]: ... in any case, if we keep the emulate-tpm I'm glad that we have a static seed here! 👍🏼 
We likely will force the machine to re-register after reboot (to update labels and annotation, in particular the IP address) and with random tpm seed the machine would just present itself (and so register at the operator) as a new one.

File: docs/partials/_quickstart-prereqs.md (hunk 3 above)
Comment (nunix) [reply docs/partials/_quickstart-prereqs.md:3]: Will be covered in a follow-up PR

File: docs/images/quickstart-ui-elemental-plugin-menu.png
Comment (nunix) [reply docs/images/quickstart-ui-elemental-plugin-menu.png:1]: Will be covered in a follow-up PR

File: docs/quickstart-ui.md (hunk 4 above)
Comment (nunix) [reply docs/quickstart-ui.md:113]: Will be covered in a follow-up PR

File: docs/quickstart-ui.md (hunk 5 above)
Comment (nunix) [reply docs/quickstart-ui.md:115]: Will be covered in a follow-up PR

**Pull request merged**